CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
+
+## Prerequisites
+
+## Build actors and providers
+
+From the top-level **directory** build with `make`. This should complete
+without errors.
+
+## Configuration
+
+Update paths in file `deploy/env` to match your development environment.
```

Comment:
I think a lot of the stuff in `env` is outdated, but I haven't confirmed

Reply:
looked through and is yankable. The other config settings we should pass through via wash and not these scripts.